### PR TITLE
Diurnal flux mods.

### PR DESCRIPTION
**Pull request by dabail10**

### CODE REVIEW

**Comment by jedwards4b** [comment]
In file shr_flux_mct.F90 lines 876-815 are repeated in lines 1201-1210 - can this be in one place?

**Comment by dabail10** [comment]
Not sure if you mean shr_flux_mod or seq_flux_mct here ...

Dave

On Thu, Jun 25, 2015 at 12:40 PM, jedwards4b notifications@github.com
wrote:

> In file shr_flux_mct.F90 lines 876-815 are repeated in lines 1201-1210 -
> can this be in one place?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CESM-Development/cime/pull/33#issuecomment-115358571.

## 

David A Bailey
 email: dbailey_at_ucar.edu
National Center for Atmospheric Research   phone: 303-497-1737
PO Box 3000
fax  : 303-497-1700
Boulder, CO 80307-3000
                   http://www.cgd.ucar.edu/staff/dbailey

**Comment by jedwards4b** [comment]
driver_cpl/driver/seq_flux_mct.F90

**Comment by dabail10** [comment]
These are in two different subroutines. The one is set up to do this on the
"exchange" grids and the other is for standard grids.

Dave

On Thu, Jun 25, 2015 at 2:44 PM, jedwards4b notifications@github.com
wrote:

> driver_cpl/driver/seq_flux_mct.F90
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CESM-Development/cime/pull/33#issuecomment-115391615.

## 

David A Bailey
 email: dbailey_at_ucar.edu
National Center for Atmospheric Research   phone: 303-497-1737
PO Box 3000
fax  : 303-497-1700
Boulder, CO 80307-3000
                   http://www.cgd.ucar.edu/staff/dbailey

**Comment by jedwards4b** [comment]
Yes but all of those lines are the same and it would be better to create another subroutine to set the values instead of risking someone changing a value in one place and missing the other.   Will there ever be a case in which they should be different?

File: share/csm_share/shr/shr_flux_mod.F90 [hunk 1]
```diff
@@ -355,6 +356,697 @@ real(R8)    :: psix2  ! stability function at ztref reference height
...
+   real(R8)    :: psimhu ! function: unstable part of psimh
+   real(R8)    :: psixhu ! function: unstable part of psimx
+   real(R8)    :: Umps   ! dummy arg ~ wind velocity (m/s)
+   real(R8)    :: Tk     ! dummy arg ~ temperature (K)
+   real(R8)    :: xd     ! dummy arg ~ ?
+   real(R8)    :: molvisc ! molecular viscosity
+   real(R8)    :: molPr   ! molecular Prandtl number 
+
+
+   qsat(Tk)   = 640380.0_R8 / exp(5107.4_R8/Tk)
```
Comment (quantheory) [inline share/csm_share/shr/shr_flux_mod.F90:587]: This is not a new issue by any stretch, but given that we've been trying to consolidate saturation calculations across CESM, it's unfortunate to have these one-off `qsat` implementations being not only kept but copied around.

**Comment by quantheory** [comment]
Just as an FYI, it's not necessary to close the isuse and open a new one when you make changes. You can just push your changes to GitHub, and this request will be updated automatically to the head of your branch. Then you can add a comment here describing what you've changed in the request.

File: share/csm_share/shr/shr_flux_mod.F90 (hunk 1 above)
Comment (jedwards4b) [inline share/csm_share/shr/shr_flux_mod.F90:587]: Thanks for catching that.

**Pull request closed without merging**